src/plater/bitmap.rs

```diff
@@ -68,6 +81,38 @@ impl Bitmap {
         }
     }
 
+    pub fn rotate_90_clockwise(&self) -> Self {
```

Comment:
Is it worth building a special case for 180 degree rotations as well? Since that's essentially equivalent to just reversing the bitmap data

Reply:
I would prefer to not build it as I think it introduces a maintenance burden. The main reason I reused the rotation functions was to ensure that the logic stayed consistent across the implementations. I think the best argument for building a special function is performance, but this isn't a bottleneck in my experience.